tests/test_component.py

```diff
@@ -0,0 +1,94 @@
+# License: MIT
+# Copyright © 2022 Frequenz Energy-as-a-Service GmbH
```

Comment:
The year should 2024 in all the new files.

Reply:
I think we should preserve the original publication date, this file is copied from the SDK which was published in 2022.

Reply:
I think we should preserve the original publication date, this file is copied from the SDK which was published in 2022.